apps/discord-bot/src/commands/leaderboards/base.leaderboard-command.tsx

```diff
@@ -52,6 +56,7 @@ export interface CreateLeaderboardOptions {
   getLeaderboard: GetLeaderboard;
   field: string;
   getLeaderboardDataIcon?: GetLeaderboardDataIcon;
+  time?: HistoricalType | undefined;
```

Comment:
```suggestion
  time?: HistoricalType;
```

I would also use the new enum like I suggested above